packages/components/Callout/src/Callout.types.ts

```diff
@@ -133,6 +132,7 @@ export interface ICalloutProps extends IViewProps, ICalloutTokens {
    * this callback is most appropriate for components strictly controlling focus.
    *
    * restoreFocusEvent.nativeEvent.containsFocus is true if the Callout had focus while being dismissed.
+   * @platform win32
```

Comment:
While I understand the want to put a note like so, I'm inclined to not for a few reasons. 

```
* Evaluations pending for platforms:
* iOS, Android
* Implementations pending for platforms:
* macOS, Windows
*
* @platform win32
*/
```

The first is some of these props are indeed win32 only, and will not be supported on other platforms (macOS won't support shouldFocusOnContainer, as an example). The other is there are no current active plans to evaluate iOS/android/windows support for Callout, so leaving such a note feels disingenuous. Moreso because this control feels very desktop specific, and may likely never come to mobile (iOS is questionable because iPad).

I'd rather have the repo reflect the current state of the world, which is these props are only supported on win32, and change the message once we add support for another platform.

Reply:
ah we should open items to remove them from the shared API then and make it win32 specific :) can move them to common if/when they are. We do have transient UI on mobile but definitely less of a use case

Reply:
Fwiw, I think Callout will eventually end up on mobile, we just have no plans to implement that. Drawer, for example, will definitely be made first.